packages/firestore/src/core/event_manager.ts

```diff
@@ -45,6 +45,10 @@ export interface Observer<T> {
  * EventManager is responsible for mapping queries to query event emitters.
  * It handles "fan-out". -- Identical queries will re-use the same watch on the
  * backend.
+ *
+ * PORTING NOTE: On Web, EventManager requires a call to `subscribe()` to
```

Comment:
This comment is no longer accurate. Instead, the onListen and onUnlisten handlers need to be assigned.

Reply:
Updated